app/models/email_volume/taxon_weekly_email_volume.rb

```diff
@@ -1,8 +1,8 @@
 module EmailVolume
   class TaxonWeeklyEmailVolume
-    HIGH = "40 - 60".freeze
-    MEDIUM = "0 - 20".freeze
-    LOW = "0 - 5".freeze
+    HIGH = "40 to 60".freeze
+    MEDIUM = "0 to 20".freeze
+    LOW = "0 to 5".freeze
```

Comment:
Nit: Could these be moved into the module? Then we could share these constants between `TaxonWeeklyEmailVolume` and `OrganisationWeeklyEmailVolume`?

Reply:
@theseanything  Yeah it did cross my mind to reduce duplication here, but I ended up not, because it's accidental duplication, i.e. the values _happen_ to be the same, but there's no fundamental reason (that I know of) why taxon and organisation email volumes would always be identical.

Happy to hear your thoughts on this.

Reply:
Ah yeah, I guess we could see different volumes, your right! Probably safer to leave, them so its easier to tweak later.